tools/cli-bootstrap/dockerfiles/debian8/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:8.0
```

Comment:
Actually, since there is more feedback than I anticipated, I will probably update this PR tomorrow with the intended Docker scripts; so, stay tuned.